### PR TITLE
stem-1.3.0 added

### PR DESCRIPTION
**Pull request by sammyshj**
The stem-1.3.0 has been added to the homebrew formula.
Details about the package can be seen at https://stem.torproject.org/

### CODE REVIEW

File: Library/Formula/stem.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  def install
+    if build.with? "pycrypto"
+      resource("pycrypto").stage do
+        system "python", *Language::Python.setup_install_args(prefix)
+      end
+    end
+    system "python", *Language::Python.setup_install_args(prefix)
+  end
+
+  test do
```
Comment (mikemcquaid) [inline Library/Formula/stem.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/stem.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class Stem < Formula
+  homepage "https://stem.torproject.org/"
+  url "https://pypi.python.org/packages/source/s/stem/stem-1.3.0.tar.bz2"
+  sha256 "770e370156e0e92a9862e4670ee3f4ac385742006e578608528ee16cbab9d416"
+
+  depends_on :python => :recommended
```
Comment (mikemcquaid) [inline Library/Formula/stem.rb]: `depends_on :python if MacOS.version <= :snow_leopard`

**Comment by sammyshj** [comment]
@mikemcquaid I have made the required changes. Please have a look at it and let me know if it is ready to be merged.

**Comment by mikemcquaid** [comment]
Oh, is this just a library? We don't really include Python libraries in Homebrew as `pip` does a better job.

**Comment by sammyshj** [comment]
Yes, this is just a python module.
This was needed so that people who don't have pip can make use of Homebrew and install the module.
The ticket for this can be seen here https://trac.torproject.org/projects/tor/ticket/8588
Please take this into consideration. Thanks

**Comment by mikemcquaid** [comment]
Everyone on OSX has `easy_install` available so they can use that or install `pip`. Sorry, will pass on this as we don't package Python libraries.

**Pull request closed without merging**